### PR TITLE
Improve documentation week 15

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: doc/use_with_siematic.rst [hunk 1]
```diff
@@ -344,6 +344,26 @@ Replace a device or a PLC
 To replace an IO-device or an IO-controller, right-click on it in the left
 side menu and select "Change device". Follow the wizard.
 
+Reload an GSDML file
+--------------------
+When the GSDML file is updated and needs to be reloaded in the Siemens environment:
+
+1. Delete all devices in your project that are type which GSDML shall be changed.
```
Comment (olbjo) [inline doc/use_with_siematic.rst:351]: "are based on the GSDML"  ?

File: doc/use_with_siematic.rst [hunk 2]
```diff
@@ -344,6 +344,26 @@ Replace a device or a PLC
...
+2. Check the "Force" table.
+
+   * Stop forcing all values.
+   * Delete entry in table.
```
Comment (olbjo) [inline doc/use_with_siematic.rst:355]: entries

**Review by olbjo — APPROVED** [review]
👍 
Minor suggestions only, in the part I wrote myself..

File: doc/use_with_siematic.rst (hunk 1 above)
Comment (pyhys) [reply doc/use_with_siematic.rst:351]: Done

File: doc/use_with_siematic.rst (hunk 2 above)
Comment (pyhys) [reply doc/use_with_siematic.rst:355]: Done

**Pull request merged**